pages/button/button.html

```diff
@@ -10,7 +10,7 @@
   <title>starUI🌟 | Button</title>
 </head>
 <body>
-  <div class="body-center">
+  <div class="body-center" id="body">
```

Comment:
I personally don't like to use keywords for ids and classes. Example <body> is a tag and id is also "body" here. I add some other word with the keyword to make it distinctive.

Reply:
> I personally don't like to use keywords for ids and classes. Example is a tag and id is also "body" here. I add some other word with the keyword to make it distinctive.

I will resolve these issues Ruchi thank you for pointing them out.